css/styles.css

```diff
@@ -50,13 +50,64 @@ body {
   font-weight: 700;
 }
 
-.navigation-bar {
+.hamburger {
+  position: relative;
+  z-index: 1000;
+  top: 21%;
+  right: 1rem;
+  padding: 4px;
+  cursor: grabbing;
+  margin-bottom: 2rem;
+}
+
+.nav-menu {
```

Comment:
- [OPTIONAL] What about preventing the sub-background from scrolling when the navbar is open? You can use a CSS property `overflow: hidden;` to do that, which will make your mobile menu correspond to the original design.